### ISSUE TITLE
Maven Conflict

### ISSUE THREAD

**Issue by AdvancedImagingUTSW** [imagej/pyimagej#68]
Hello,
I created a Conda environment from the environment.yml file provided on pyimagej.  Upon trying to launch ImageJ from within this python environment failed, due to a maven error.  

>>> import imagej
>>> ij = imagej.init()
Error in `C:\Users\MicroscopyInnovation\Anaconda3\envs\imagej\Library\bin\mvn.CMD -B -f C:\Users\MicroscopyInnovation\.jgo\net.imagej\imagej\RELEASE+net.imglib-imglib2-imglyb-0.3.0\pom.xml dependency:resolve': 1

I suspect that it has something to do with a proxy, which in the past, I had to set up manually for Maven.  However, this pom.xml file seems to be created anew each time I launch ImageJ.  This error happens if I try to launch a local version of ImageJ via the command ij = imagej.init('/myPath/Fiji.app'), or the latest one via the command ij = imagej.init()

Any suggestions?  Windows 10 machine.

Thank you,
Kevin

**Comment by ctrueden** [comment]
@AdvancedImagingUTSW You can get more details by running that command yourself from the command line:
```
C:\Users\MicroscopyInnovation\Anaconda3\envs\imagej\Library\bin\mvn.CMD -B -f C:\Users\MicroscopyInnovation.jgo\net.imagej\imagej\RELEASE+net.imglib-imglib2-imglyb-0.3.0\pom.xml dependency:resolve
```
And you'll see a lot of information, including the ultimate reason why the command fails. This will be helpful for diagnosing if it's a proxy issue, versus some other cause.

I do notice that `C:\Users\MicroscopyInnovation.jgo\net.imagej\imagej\RELEASE+net.imglib-imglib2-imglyb-0.3.0\pom.xml` is the wrong path—it should be `C:\Users\MicroscopyInnovation\.jgo\net.imagej\imagej\RELEASE+net.imglib-imglib2-imglyb-0.3.0\pom.xml` with a backslash between your user folder name and the `.jgo` cache folder. That might be the source of the problem here?

Finally, about proxying with Maven: that is configured in `.mvn/settings.xml`. If you have a `settings.xml` file there, check it to see what's present. You could try renaming it before running pyimagej again.

**Comment by FengZhiheng** [comment]
> Hello,
> I created a Conda environment from the environment.yml file provided on pyimagej. Upon trying to launch ImageJ from within this python environment failed, due to a maven error.
> 
> > > > import imagej
> > > > ij = imagej.init()
> > > > Error in `C:\Users\MicroscopyInnovation\Anaconda3\envs\imagej\Library\bin\mvn.CMD -B -f C:\Users\MicroscopyInnovation.jgo\net.imagej\imagej\RELEASE+net.imglib-imglib2-imglyb-0.3.0\pom.xml dependency:resolve': 1
> 
> I suspect that it has something to do with a proxy, which in the past, I had to set up manually for Maven. However, this pom.xml file seems to be created anew each time I launch ImageJ. This error happens if I try to launch a local version of ImageJ via the command ij = imagej.init('/myPath/Fiji.app'), or the latest one via the command ij = imagej.init()
> 
> Any suggestions? Windows 10 machine.
> 
> Thank you,
> Kevin

I have the same problem

**Comment by FengZhiheng** [comment]
> > Hello,
> > I created a Conda environment from the environment.yml file provided on pyimagej. Upon trying to launch ImageJ from within this python environment failed, due to a maven error.
> > > > > import imagej
> > > > > ij = imagej.init()
> > > > > Error in `C:\Users\MicroscopyInnovation\Anaconda3\envs\imagej\Library\bin\mvn.CMD -B -f C:\Users\MicroscopyInnovation.jgo\net.imagej\imagej\RELEASE+net.imglib-imglib2-imglyb-0.3.0\pom.xml dependency:resolve': 1
> > 
> > 
> > I suspect that it has something to do with a proxy, which in the past, I had to set up manually for Maven. However, this pom.xml file seems to be created anew each time I launch ImageJ. This error happens if I try to launch a local version of ImageJ via the command ij = imagej.init('/myPath/Fiji.app'), or the latest one via the command ij = imagej.init()
> > Any suggestions? Windows 10 machine.
> > Thank you,
> > Kevin
> 
> I have the same problem. I tried to run that command myself and I got this:
> ![image](https://user-images.githubusercontent.com/27288257/88878036-8eec6200-d259-11ea-8ea3-1c6a556cae4c.png)

I cannot load jars

https://maven.scijava.org/#nexus-search;gav~net.imagej~imagej~~~



![Snipaste_2020-07-30_15-32-40](https://user-images.githubusercontent.com/27288257/88894099-f4e8e180-d279-11ea-83d5-a24c3d320598.jpg)

**Comment by ctrueden** [comment]
@FengZhiheng Sorry for the late reply. Looks like the SciJava Maven repository had a transient error back then. Are you still having problems? If so, please reopen.

@AdvancedImagingUTSW Since I didn't hear back, I'm going to assume you either figured something out, or gave up? Either way, if it still doesn't work, and you want to troubleshoot further, please reopen this after testing with the newly released pyimagej 1.0.0 and friends.
